client/lib/feed-stream-store/feed-stream.js

```diff
@@ -240,10 +240,10 @@ assign( FeedStream.prototype, {
 		return date;
 	},
 
-	hasRecentError: function() {
+	hasRecentError: function( errorType ) {
 		var aMinuteAgo = Date.now() - ( 60 * 1000 );
 		return this.errors.some( function( error ) {
-			return error.timestamp && error.timestamp > aMinuteAgo;
+			return ( error.timestamp && error.timestamp > aMinuteAgo ) && ( !errorType || errorType === error.error );
```

Comment:
Can we pop a space in after the exclamation mark? `! errorType`

It'd be good to add a jsdoc block to the hasRecentError() method too, to explain what we expect from the errorType param.